### ISSUE TITLE
Alert on COVID FAQ page not rendering correctly in Drupal preview

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#7552]
## What happened?
The "How do I talk to someone right now?" alert on the COVID FAQ page is rendering as a gray box on the Drupal preview in prod. It is showing correctly on the live site but after copy changes were made to other parts of the page, the gray box started showing. 

![Screen Shot 2020-04-01 at 11.12.34 AM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/e993db92-d19a-4a35-b375-22482a1f24a9)

## Specs:
- Device: Macbook Pro
- Browser: Chrome
- Test User _(if applicable)_:

## Steps to Reproduce
Drupal Node ID: http://preview-prod.vfs.va.gov/preview?nodeId=3014

## Desired behavior
Alert should show up correctly.

## Acceptance Criteria
- [ ] Alert working in Preview and live site

**Comment by MickinSahni** [comment]
2020-04-06 PW Standup
- Can be validated by @RLHecht

**Comment by RLHecht** [comment]
@MickinSahni We are holding off on putting this alert back on the FAQ page until later in the week, so I can't test yet whether it's working in preview. Or I can put it back in as a draft, test it and then take it out, but it probably makes more sense to just wait on it.

**Comment by MickinSahni** [comment]
> @MickinSahni We are holding off on putting this alert back on the FAQ page until later in the week, so I can't test yet whether it's working in preview. Or I can put it back in as a draft, test it and then take it out, but it probably makes more sense to just wait on it.

👍 we'll add to Sprint 19 @RLHecht

**Comment by RLHecht** [comment]
@MickinSahni We are still holding off on putting this back into the page.

**Comment by MickinSahni** [comment]
2020-04-13 PW Standup
- On-hold until we get the go-ahead

**Comment by stephorkandatx** [comment]
2020-04 20: PW Standup

This work is still on hold, and has been moved to Blocked until we can proceed

**Comment by stephorkandatx** [comment]
We are keeping this widget off the FAQ page, however the template update may not have been validated yet. 

Can we validate if that's addressed - if so, we could close this ticket out.
If the template issue remains - will need to look at again.
